src/org/omegat/util/Language.java

```diff
@@ -774,26 +774,11 @@ public boolean isSameCountryLanguage(String target) {
         return isSameCountryLanguage(new Language(target));
     }
 
-    /**
-     * Check if locale is Right-To-Left oriented.
-     * @return true if locale is Right-To-Left oriented.
-     */
-    public static boolean localeIsRTL() {
-        String language = Locale.getDefault().getLanguage().toLowerCase(Locale.ENGLISH);
-        return isRTL(language);
+    public static String getLowerCaseLanguageFromLocale() {
+        return Locale.getDefault().getLanguage().toLowerCase(Locale.ENGLISH);
     }
 
-    /**
-     * Check if language is Right-To-Left oriented.
-     *
-     * @param language
-     *            ISO-639-2 language code
-     * @return true if language is RTL
-     */
-    public static boolean isRTL(final String language) {
```

Comment:
To keep backward compatibility, please leave the method.
you can add `@deprecated`